src/request.rs

```diff
@@ -1,17 +1,26 @@
 use napi::bindgen_prelude::{BigInt, Buffer};
-use scylla::frame::{
-    response::result::CqlValue,
-    value::{Counter, CqlTimeuuid},
+use scylla::{
+    frame::{
+        response::result::CqlValue,
+        value::{Counter, CqlTimeuuid},
+    },
+    prepared_statement::PreparedStatement,
 };
 
 use crate::{
+    result::{map_column_type_to_cql_type, CqlType},
     types::uuid::UuidWrapper,
     utils::{bigint_to_i64, js_error},
 };
 
 #[napi]
 pub struct QueryParameterWrapper {
-    parameter: CqlValue,
+    pub(crate) parameter: CqlValue,
+}
+
+#[napi]
+pub struct PreparedStatementWrapper {
+    pub(crate) query: PreparedStatement,
 }
```

Comment:
When introducing public structs like these, let's try to write at least minimal docstrings, so that reading the code is easier. Just by hovering the type's name we will get a short info what it is for.